### DISCUSSION TITLE
How to summon Android system onscreen navigation buttons?

### DISCUSSION THREAD

**Discussion by gingerbeardman** [koreader/koreader#10753] [Q&A] [answered]
My Android device has on-screen back/home/switcher buttons. 

Is there a way to make them show up in KOReader? 

Or alternatives,  such as gestures? I have looked, but couldn't find any. 

Exiting and relaunching KOReader each time I switch to another app seems too much.

**Comment by gingerbeardman** [comment, marked as answer]
Swipe up onto the bottom edge of the screen and look carefully, as the onscreen system buttons will probably be overlaid on the bottom menu.

**Comment by mergen3107** [comment]
You can disable “Activate menu by swipe” and then only use taps at top/bottom for KOReader menus, and swipe for Notifications/Navbar

**Reply by gingerbeardman** [reply]
I'll do that, thanks.
